core/src/test/java/apoc/spatial/GeocodeTest.java

```diff
@@ -37,6 +42,38 @@ public void setUp() throws Exception {
         assumeRunningInCI();
```

Comment:
Sorry, but again, I wouldn't have tests that don't run in the CI nor locally and we don't have any instructions to run. To the best of my knowledge the OPENCAGE_KEY is not set in the CIs (both GitHub Actions or TeamCity). Are there any plans to set that there?

Reply:
Ok sorry, I thought you just wanted me to add mocked tests.

However, I don't have access to TeamCity and therefore I don't know how to set / write the instructions to activate the `OPENCAGE_KEY`.

In any case, I have now added the mocked tests (`SpatialTest.java`).
So, could I just leave (in `GeocodeTest.java`) the osm tests, which don't require a key, and revert the opencage ones?

Although, being hard-coded present:
```
apocConfig().setProperty("apoc.spatial.geocode.opencage.key", "<YOUR_API_KEY>");
```
and 
```
Assume.assumeTrue(!"<YOUR_API_KEY>".equals(apocConfig().getString(Geocode.PREFIX +"." + provider + ".key")));
```
the tests regarding opencage are not performed in any case.

Reply:
First of all, so sorry for the delay reviewing.

The problem for me is precisely having tests in that are never run in the CI. There's an ongoing effort by @AzuObs to enable as many tests as possible in the CI, so I don't think this aligns with that. I'd prefer to have only mocked tests for this feature if you don't mind.

Reply:
I updated the pr by deleting the additional ignored tests. @ncordon

Reply:
Thanks 👍